src/tree/EnvironmentTreeItem.ts

```diff
@@ -91,7 +91,19 @@ export class EnvironmentTreeItem extends AzureParentTreeItem implements IAzureRe
         return treeUtils.getIconPath('Azure-Static-Apps-Environment');
     }
 
-    public async loadMoreChildrenImpl(_clearCache: boolean, _context: IActionContext): Promise<AzExtParentTreeItem[]> {
+    public async loadMoreChildrenImpl(_clearCache: boolean, _context: IActionContext): Promise<AzExtTreeItem[]> {
+        const client: WebSiteManagementClient = createAzureClient(this.root, WebSiteManagementClient);
+        const functions: WebSiteManagementModels.StaticSiteFunctionOverviewCollection = await client.staticSites.listStaticSiteBuildFunctions(this.parent.resourceGroup, this.parent.name, this.buildId);
+        client.staticSites.listStatuc;
+        if (functions.length === 0) {
+            return [this.actionsTreeItem, new GenericTreeItem(this, {
+                label: localize('noFunctions', 'Learn more about Functions in Azure Static Web Apps...'),
+                contextValue: 'noFunctions',
+                commandId: 'staticWebApps.showFunctionsDocumentation',
+                iconPath: new ThemeIcon('question')
```

Comment:
Other places use the little book to indicate documentation

![Screen Shot 2020-11-18 at 11 21 49 AM](https://user-images.githubusercontent.com/11282622/99577523-58655f00-2990-11eb-9aa4-6c637991318c.png)